jetbrains-core/tst/software/aws/toolkits/jetbrains/services/codewhisperer/CodeWhispererTelemetryTest.kt

```diff
@@ -380,6 +380,36 @@ class CodeWhispererTelemetryTest : CodeWhispererTestBase() {
         )
     }
 
+    @Test
+    fun `test showing IntelliSense after triggering CodeWhisperer will send userDecision events of state Discard`() {
+        val codewhispererServiceSpy = spy(codewhispererService)
+        codewhispererServiceSpy.stub {
+            onGeneric {
+                canDoInvocation(any(), any())
+            } doAnswer {
+                true
+            }
+        }
+        ApplicationManager.getApplication().replaceService(CodeWhispererService::class.java, codewhispererServiceSpy, disposableRule.disposable)
+        popupManagerSpy.stub {
+            onGeneric {
+                hasConflictingPopups(any())
+            } doAnswer {
+                true
+            }
+        }
+        invokeCodeWhispererService()
+
+        runInEdtAndWait {
+            val metricCaptor = argumentCaptor<MetricEvent>()
+            verify(batcher, atLeastOnce()).enqueue(metricCaptor.capture())
+            assertEventsContainsFieldsAndCount(
+                metricCaptor.allValues, userDecision, pythonResponse.recommendations().size,
```

Comment:
just a nit-pick Andrew, should we make it multi-lines

Reply:
yes, each value should be on its on line if you're breaking it up like this. not sure why our lint rule isn't catching this

Reply:
good catch